workspace/utilities/lib/util-nav-link-attr.xsl

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+
+<!-- COMPONENT: util-nav-link-attr -->
+	<xsl:template name="util-nav-link-attr">
+		<xsl:param name="label" select="label" />
+		<xsl:param name="ga-cat" select="'menu-nav'" />
+		<xsl:param name="ga-label" select="concat($url-language, '/', $label)" />
```

Comment:
Please add another parameter: `ga-action` . Its default value can be nothing.

Reply:
Done!

Reply:
:+1: